docs/source/1.0/spec/aws/aws-cloudformation.rst

```diff
@@ -0,0 +1,672 @@
+        }
+
+.. _aws.cloudformation#additionalIdentifier-trait:
+
+-------------------------------------------------
+``aws.cloudformation#additionalIdentifier`` trait
+-------------------------------------------------
+
+Summary
+    Indicates that the CloudFormation property generated from this member is an
+    additional identifier for the resource.
+Trait selector
+    ``structure > :test(member > string)``
+
+    *Any structure member that targets a string*
+Value type
+    Annotation trait
+Conflicts with
+    :ref:`aws.cloudformation#writeOnlyProperty-trait`
+
+The following example defines a CloudFormation resource that has the
+``fooAlias`` property as an additional identifier:
+
+.. tabs::
+
+    .. code-tab:: smithy
+
+        namespace smithy.example
+
+        use aws.cloudformation#additionalIdentifier
+        use aws.cloudformation#resource
+
+        @resource
+        resource Foo {
+            identifiers: {
+                fooId: String,
+            },
+            get: GetFoo,
+        }
+
+        operation GetFoo {
+            input: GetFooRequest,
+        }
+
+        structure GetFooRequest {
+            @required
+            fooId: String,
+
+            @additionalIdentifier
+            fooAlias: String,
+        }
+
+
+.. _aws.cloudformation#propertyName-trait:
+
+-----------------------------------------
+``aws.cloudformation#propertyName`` trait
+-----------------------------------------
+
+Summary
+    The propertyName trait allows a CloudFormation resource property name to
+    differ from a structure member name used in the model.
+Trait selector
+    ``structure > member``
+
+    *Any structure member*
+Value type
+    ``string``
+
+Given the following structure definition that is derived in to a CloudFormation
+resource:
+
+.. tabs::
+
+    .. code-tab:: smithy
+
+        namespace smithy.example
+
+        use aws.cloudformation#propertyName
+
+        structure AdditionalFooProperties {
+            bar: String,
+
+            @propertyName("Tags")
+            tagList: TagList,
+        }
+
+the CloudFormation resource would have the following property names derived
+from it:
+
+::
+
+    "bar"
+    "Tags"
+
+
+.. _aws.cloudformation#excludeProperty-trait:
+
+--------------------------------------------
+``aws.cloudformation#excludeProperty`` trait
+--------------------------------------------
+
+Summary
+    Indicates that structure member should not be included in generated
+    CloudFormation resource definitions.
+Trait selector
+    ``structure > member``
+
+    *Any structure member*
+Value type
+    Annotation trait
+Conflicts with
+    :ref:`aws.cloudformation#additionalIdentifier-trait`,
+    :ref:`aws.cloudformation#createOnlyProperty-trait`,
+    :ref:`aws.cloudformation#mutableProperty-trait`,
+    :ref:`aws.cloudformation#readOnlyProperty-trait`,
+    :ref:`aws.cloudformation#writeOnlyProperty-trait`
+
+The following example defines a CloudFormation resource that has the derivable
```

Comment:
"has the derivable ``responseCode`` property excluded" -> "excludes the ``responseCode`` property"

Reply:
Can you link to the CFN specs for a what a property is?